src/zcl_aff_writer_xslt.clas.testclasses.abap

```diff
@@ -933,7 +935,7 @@ CLASS ltcl_integration_test IMPLEMENTATION.
     DATA(st_name) = CONV progname( c_xslt_prefix && st_execution_counter ).
     st_name = |{ st_name WIDTH = 30 PAD = '=' }XT|.
```

Comment:
there is also this line, which I guess is the same constant?

Reply:
Sorry, I don't see what you mean. This line concatenates the st name with some '=' und the letters 'XT'

Reply:
the newly defined constant contains "XT"..., can it be used instead of this hardcoded string?

Reply:
Ah yeah, sure. Will be fixed in next commit